public/bellomos.html

```diff
@@ -1,41 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-      crossorigin=""
-    />
-    <script
-      src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
-      integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
-      crossorigin=""
-    ></script>
-    <link href="index.css" rel="stylesheet" />
-  </head>
-  <body>
-    <div class="Navbar">
-      <a href="ramuntos.html">Ramunto's</a>
-      <a href="johnny-boys.html">Johnny Boy's Pancake House</a>
-      <a href="orientaste.html">Orientaste</a>
-      <a href="bellomos.html">Bellomo's Market</a>
-      <a href="rollin-rooster.html">The Rollin' Rooster</a>
-      <a href="taco-bell.html">Taco Bell</a>
-    </div>
-    <h1 id="title">Bellomo's Market</h1>
-    <p id="info">
-      Bellomo's Market<br />
-      31 Forest St. Rutland, VT<br />
-      (802)775-1632<br />
-      Hours: M-F: 7AM-8PM<br />
-      Sat: 9AM-6PM Sun: Closed<br />
-      Notes: Small market with a deli. Best club sandwhiches in town.
-    </p>
```

Comment:
We could also generate this information with our fetch that we did previously, and generate the name, address, phone number and everything else that's included with our json objects that we've built and should be fetching for. 

With doing it this way, we can accomplish all of this with just one page that gets generated depending on the fetch that we are looking for.